documentation/practices.md

```diff
@@ -80,5 +80,37 @@ Configuration properties for content URLs are:
 
 For any other CMS, you must update the configurations to reflect the public URLs of the files being retrieved by the particular content provider.
 
+## Indi Brand Loyalty Example
+
+The React PWA Reference Storefront is pre-configured for integration with Indi, with a custom component created to interact with the various widgets Indi may display. For more information on Indi, visit their official [website](https://indi.com/).
+
+The Indi component available is a stand-alone component created solely to fetch the `indi-embed` libraries, then display the corresponding elements that were passed into the component as a structured list.
+
+The required parameters of this component are as follows:
+* `render`: A list of the elements to display from Indi. Example: `render={['carousel', 'brand', 'product']}`. Note the correct configurations must be provided for the element to render correctly.
+* `configuration`: The structured object consisting of the configurations to provide to the Indi components. These are defined in `ep.config.json` in the `indi` element.
+* `keywords`: A string of the keywords for the product in which the Indi component is displayed for.
+
+Static strings for the Indi component have been localized in the Reference Storefront and are overrided on a per-usage basis for the Indi component. This will ensure the component displays the correct text in the storefront based on the locale which a customer chooses during their shopping flow.
```

Comment:
```suggestion
Static strings for the Indi component are localized in the Reference Storefront and are overridden whenever the Indi component is used. With this setting, the component displays the storefront in the locale that a customer chooses.
```

Reply:
Why is this section here? Why are we suddenly talking about Static strings here when in the next section we are going to explain `configuration`?

Reply:
I wanted to call out that we provide localization options for the strings a developer would normally pass to the Indi library to display the component. This allows for other languages to be used for the component dynamically rather than hardcoding the strings through configurations.

Reply:
Do you want to have a heading here? @shaunmaharaj